web/src/lib/components/shared-components/album-selection-modal.svelte

```diff
@@ -44,7 +44,7 @@
   };
 </script>
 
-<BaseModal on:close={() => dispatch('close')}>
+<BaseModal on:close on:escape>
```

Comment:
I don't think we use `on:escape` here, correct? Which event's reaction does it do?

Reply:
Ah, so this propagates the event out for this component?

Reply:
Yes exactly!